cla_backend/apps/cla_butler/management/commands/monitor_multiple_outcome_codes.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+import logging
+from django.core.management.base import BaseCommand
+from django.db.models import Count, Max, Min
+from django.utils.timezone import now
+from cla_butler.stack import is_first_instance, InstanceNotInAsgException, StackException
+from cla_eventlog.models import Log
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'LGA-294 specific monitoring command. Alert when multiple outcome codes ' \
+           'that should only occur once are found for today (since 00:00)'
+
+    def handle(self, *args, **options):
+        if self.should_run_housekeeping(**options):
+            self.check_for_multiple_outcome_codes()
+        else:
+            logger.debug('LGA-294 Skip check_for_multiple_outcome_codes: running on secondary instance')
+
+    @staticmethod
+    def check_for_multiple_outcome_codes():
+        # Outcome codes defined to appear only once on a case:
+        # https://docs.google.com/spreadsheets/d/1hN64bA_H2a_0eC_5-k-0IY2-RKbCor2VGflp1ykQGa0/
+        start_of_today = now().replace(hour=0, minute=0, second=0, microsecond=0)
+        once_only_codes = ['PCB', 'COPE', 'DUPL', 'MRNB', 'NCOE', 'DESP', 'DECL', 'MRCC', 'NRES', 'CPTA',
```

Comment:
🙋‍♂️ Would it be worth extending the definitions of these codes in `cla_backend/apps/call_centre/events.py` with `unique: True`? (not in this pull request)